sw/otbn/code-snippets/solinas384.s

```diff
@@ -207,9 +212,11 @@ solinas384:
 .section .text.start
 .globl wrap_solinas384
 wrap_solinas384:
-  /* Initialize all-zero register. */
+  /* Initialise WDRs before using them */
   bn.xor w31, w31, w31
-
+  bn.xor w14, w14, w14
+  bn.xor w15, w15, w15
+  bn.xor w16, w16, w16
```

Comment:
I'd suggest leaving the all-zero register and the initialization code separate, like:
```
/* Initialize all-zero register. */
bn.xor  w31, w31, w31

/* Initialize WDRs before using them. */
bn.xor  w14, w14, w14
bn.xor  w15, w15, w15
bn.xor  w16, w16, w16
```
Conceptually, they're a bit different, and the `w31` register is consistently initialized to zero all over OTBN code.